html/global_attributes.json

```
@@ -150,6 +150,43 @@
             }
           }
         },
+        "one-time-code": {
+          "__compat": {
+            "description": "<code>new-password</code> value",
+            "support": {
+              "chrome": {
+                "version_added": "93"
+              },
+              "chrome_android": {
+                "version_added": "84"
+              },
+              "edge": "mirror",
+              "firefox": {
+                "version_added": false
+              },
+              "firefox_android": "mirror",
+              "ie": {
+                "version_added": false
+              },
+              "oculus": "mirror",
+              "opera": "mirror",
+              "opera_android": "mirror",
+              "safari": {
+                "version_added": "14"
```

This is in Safari? Source?

---

See the second paragraph below the heading at https://developer.chrome.com/articles/web-otp/#annotate-an-input-tag

---

Actually, it should be 12 per https://github.com/whatwg/html/issues/3745#issuecomment-395147558


```suggestion
                "version_added": "12"
```